understanding/error-prevention-legal-financial-data.html

```diff
@@ -58,160 +56,118 @@
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>For <a href="#dfn-web-page">Web pages</a> that cause <a href="#dfn-legal-commitments">legal commitments</a> or financial transactions for the user to occur, that modify or delete <a href="#dfn-user-controllable">user-controllable</a> data in data storage systems, or that submit user test responses, at least one of
-                     the following is true:
+                  <p>利用者にとって<a href="#dfn-legal-commitments">法律行為</a>もしくは金融取引が生じる、 <a href="#dfn-user-controllable">利用者が制御可能な</a>データストレージシステム上のデータを変更もしくは削除する、又は利用者が試験の解答を送信する<a href="#dfn-web-page">ウェブページ</a>では、次に挙げる事項のうち、少なくとも一つを満たしている:
                      
                   </p>
                   <dl>
                      
-                     <dt>Reversible</dt>
+                     <dt>取消</dt>
                      
-                     <dd>Submissions are reversible.</dd>
+                     <dd>送信を取り消すことができる。</dd>
                      
-                     <dt>Checked</dt>
+                     <dt>チェック</dt>
                      
-                     <dd>Data entered by the user is checked for input errors and the user is provided an opportunity
-                        to correct them.
+                     <dd>利用者が入力したデータの入力エラーがチェックされ、利用者には修正する機会が提供される。
                      </dd>
                      
-                     <dt>Confirmed</dt>
+                     <dt>確認</dt>
                      
-                     <dd>A mechanism is available for reviewing, confirming, and correcting information before
-                        finalizing the submission.
+                     <dd>送信を完了する前に、利用者が情報の見直し、確認及び修正をするメカニズムが利用できる。
                      </dd>
                      
                   </dl>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Users can avoid submitting incorrect important information.</dd>
-                  
-                  <dt>What to do</dt>
-                  <dd>Provide ways for users to confirm, correct, or reverse important submissions.</dd>
-                  
-                  <dt>Why it's important</dt>
-                  <dd>People with disabilities may be more likely to make mistakes, or not notice them.</dd> 
+                  <dt>目標</dt>
+                  <dd>利用者が誤った重要な情報を送信することを避けることができる。</dd>
+                  		         
+                  <dt>何をすればよいか</dt>
+                  <dd>利用者が重要な送信内容を確認、修正、又は取り消す方法を提供する。</dd>
+                  			      
+                  <dt>なぜそれが重要か</dt>
+                  <dd>障害のある人は間違えたり、間違いに気づかなかったりする可能性が高い。</dd> 
```

Comment:
「可能性が高い」が「気づかなかったりする」だけにかかっているのか、「間違えたり」にもかかっているのか、どちらとも取れる文になっています。ここでは両方にかかっていると思いますので、たとえばこんな感じではどうでしょうか。

```suggestion
                  <dd>障害のある人は間違いをしやすく、間違いに気づかない可能性も高い。</dd> 
```